presto-docs/src/main/sphinx/release/release-319.rst

```diff
@@ -0,0 +1,45 @@
+===========
+Release 319
+===========
+
+General Changes
+---------------
+* Fix planning failure for queries involving ``UNION`` and ``DISTINCT`` aggregates. (:issue:`1510`)
+* Fix excessive runtime when parsing expressions involving ``CASE``. (:issue:`1407`)
+* Fix fragment's output size in the output of ``EXPLAIN ANALYZE``. (:issue:`1345`)
+* Improve performance of queries against ``information_schema`` tables. (:issue:`1329`, :issue:`1308`)
+* Eliminate cross joins in some queries with coercions and complex expressions
```

Comment:
What's the user-visible impact of this? Performance improvements? Some queries that used to fail now succeed?

Reply:
Performance.